src/databases/Blueprint/avtBlueprintFileFormat.C

```diff
@@ -1135,6 +1214,16 @@ avtBlueprintFileFormat::PopulateDatabaseMetaData(avtDatabaseMetaData *md)
             const Node &n = itr.next();
             AddBlueprintMeshAndFieldMetadata(metadata, itr.name(), n);
         }
+
+        // Rewind to process all expressions *after* all fields. This
+        // helps filter expression content relative to all known meshes.
+        NodeConstIterator itr2 = m_root_node["blueprint_index"].children();
```

Comment:
I didn't know how to re-init `itr` so I was lazy here. What is the correct thing to simply restart `itr` from beginning?

Reply:
No worries, I think thats fine. 
For future ref, you can reset the iterator with `to_front(`), our you could just do
```
itr = m_root_node["blueprint_index"].children();
```
Iterators are relatively lightweight so  creating a new object vs re-init, no real consequences.